grails-app/conf/Config.groovy

```diff
@@ -277,7 +277,7 @@ grails.gorm.default.mapping = {
 
 detection {
     extract {
-        limit = 50000
+        fetchSize = 1000
```

Comment:
This is what we use for geoserver and its been working fine there.  But yes, if it performs badly in systest then its easily changed.  I did notice that delivery of the zip stream was a bit stop and start, but I think this is more to do with the buffer size used by the zipping process.  Its something we should align more with geoserver settings as well perhaps.